backend/capellacollab/projects/toolmodels/modelbadge/routes.py

```diff
@@ -35,10 +35,14 @@ async def get_model_complexity_badge(
     logger: logging.LoggerAdapter = fastapi.Depends(log.get_request_logger),
 ):
     try:
+        (
+            _,
+            model_badge,
+        ) = await git_handler.get_file_from_repository_or_artifacts(
+            "model-complexity-badge.svg", "generate-model-badge"
+        )
         return fastapi.responses.Response(
-            content=await git_handler.get_file_from_repository_or_artifacts(
-                "model-complexity-badge.svg", "generate-model-badge"
-            ),
+            content=model_badge,
```

Comment:
```suggestion
            content=git_handler.get_file_from_repository_or_artifacts(
                "model-complexity-badge.svg", "generate-model-badge"
            )[1],
```

I'd prefer this solution as it saves two variable definitions.